akka-stream-testkit/src/test/resources/reference.conf

```diff
@@ -2,7 +2,7 @@
 #
 # All stream tests should use the dedicated `akka.test.stream-dispatcher` or disable this validation by defining:
 # akka.actor.default-mailbox.mailbox-type = "akka.dispatch.UnboundedMailbox"
-akka.actor.default-mailbox.mailbox-type = "akka.stream.testkit.StreamTestDefaultMailbox"
+# akka.actor.default-mailbox.mailbox-type = "akka.stream.testkit.StreamTestDefaultMailbox"
```

Comment:
Oh, this shouldn't be pushed here - sorry.
// Since it wants the dispatcher to be the test dispatcher it disrupted my setting of io-dispatcher. I'll change that to be only overriden only in my test.